ssn/images/historical-airtemp.svg

```diff
@@ -1,4 +1,4 @@
roke-dasharray="8 8" pointer-events="stroke" style="stroke: rgb(51, 51, 51);"/><path d="M 761.54 182.91 L 765.6 174.57 L 756.49 176.33" fill="none" stroke="#333333" stroke-width="2.3" stroke-miterlimit="10" pointer-events="all" style="stroke: rgb(51, 51, 51);"/></g></g><g data-cell-id="b_WqZA4BNDlM4abTrYiu-94"><g><rect x="741" y="184" width="140" height="30" fill="none" stroke="none" pointer-events="all"/></g><g><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 142px; height: 1px; padding-top: 199px; margin-left: 740px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; width: 140px; max-height: 30px; color: #000000; "><div style="display: inline-block; font-size: 14px; font-family: &quot;Helvetica&quot;; color: light-dark(#000000, #ffffff); line-height: 1.2; pointer-events: all; font-style: italic; width: 100%; white-space: normal; word-wrap: normal; ">phenomenonTime</div></div></div></foreignObject><text x="811" y="203" fill="light-dark(#000000, #ffffff)" font-family="&quot;Helvetica&quot;" font-size="14px" text-anchor="middle" font-style="italic">phenomenonTime</text></switch></g></g></g><g data-cell-id="b_WqZA4BNDlM4abTrYiu-56"><g><rect x="546" y="370.5" width="145" height="27" fill="#ffffff" stroke="none" pointer-events="all" style="fill: light-dark(#ffffff, var(--ge-dark-color, #121212));"/></g><g><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 147px; height: 1px; padding-top: 384px; margin-left: 545px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; width: 145px; max-height: 27px; color: #000000; "><div style="display: inline-block; font-size: 14px; font-family: &quot;Helvetica&quot;; color: light-dark(#000000, #ffffff); line-height: 1.2; pointer-events: all; font-style: italic; width: 100%; white-space: normal; word-wrap: normal; ">hasFeatureOfInterest</div></div></div></foreignObject><text x="619" y="388" fill="light-dark(#000000, #ffffff)" font-family="&quot;Helvetica&quot;" font-size="14px" text-anchor="middle" font-style="italic">hasFeatureOfInterest</text></switch></g></g></g></g></g></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.drawio.com/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Text is not SVG - cannot display</text></a></switch></svg>
```

Comment:
Are the black rectangles intentional? They don't seem to be helpful.

Reply:
Can't see them

Reply:
<img width="1106" height="755" alt="Screen Shot 2025-07-11 at 04 11 31 PM" src="https://github.com/user-attachments/assets/f1c274d0-6988-480d-9034-dbc2c862d1dc" />